lib/register_client_manager.rb

```diff
@@ -2,22 +2,28 @@
 require 'in_memory_data_store'
 
 module RegistersClient
-    VERSION = '0.8.0'
+    VERSION = '0.9.0'
     class RegisterClientManager
       def initialize(config_options = {})
         @config_options = defaults.merge(config_options)
         @register_clients = {}
       end
   
       def get_register(register, phase, data_store = nil)
-        key = register + ':' + phase
+        environment_url = get_environment_url_from_phase(phase)
+        get_register_from_environment(register, environment_url, data_store)
+      end
+
+      def get_register_from_environment(register, environment_url, data_store = nil) 
```

Comment:
Is this method meant to be public, or should it be private? I'm also a little unsure of introducing this method, as opposed to keeping the logic inside of `get_register`, but happy to hear opinions on this.

Reply:
I made it public to give someone using the client library the option to use it on an environment that isn't one of the existing ones. For example, it might be one of us spinning up an environment on our local machine.

Reply:
Got it, makes sense.